src/catalog/src/system.rs

```diff
@@ -208,6 +205,29 @@ pub fn build_table_insert_request(
     )
 }
 
+pub(crate) fn build_table_deletion_request(
+    request: &DeregisterTableRequest,
+    table_id: TableId,
+) -> DeleteRequest {
+    let table_key = format_table_entry_key(&request.catalog, &request.schema, table_id);
+
+    let mut key_column_values = HashMap::with_capacity(3);
+    key_column_values.insert(
+        "entry_type".to_string(),
```

Comment:
Since these column names like "entry_type"/"key"/"timestamp" are used both in `build_insert_request` and `build_table_deletion_request`, better to extract them to constants.